### ISSUE TITLE
Checkbox: theming bug

### ISSUE THREAD

**Issue by d0gg3r** [primefaces/primereact#6423]
### Describe the bug

Incorrect display of the checkbox, additional input appears
![image](https://github.com/primefaces/primereact/assets/98053992/ca6bcf19-99ab-4d8c-a083-71bfbef056d8)

### Reproducer

https://primereact.org/checkbox/

### PrimeReact version

10.6.3

### React version

18.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

Chrome

### Steps to reproduce the behavior

1. Go to https://primereact.org/checkbox/  -> Theming Tab -> scroll to section Playground sample -> switch the checkbox

### Expected behavior

correct display of the checkbox during self-theming

**Comment by gcko** [comment]
I’m unsure of the question - i fixed the underlying issue (the missing identifier on input) and I also fixed the tailwind theme. I submitted it as a PR

Best,

Jared Scott
Founder | Variable
+1 (562) 302-8063
+886966463236
________________________________

**Comment by melloware** [comment]
Yes i think the Tailwind theme is wrong for Checkbox and Radio as well...

**Comment by melloware** [comment]
This is fixed with: https://github.com/primefaces/primereact/issues/6346
